ODataVirtualDataSourceDataProviderWorker.ts

```diff
@@ -468,6 +471,25 @@ export class ODataVirtualDataSourceDataProviderWorker extends AsyncVirtualDataSo
 		let success_: (arg1: string) => void = (res: string) => {
 			let sp: ODataSchemaProvider = new ODataSchemaProvider(res);
 			let schema = sp.getODataDataSourceSchema(this._entitySet);
+			if (this._schemaIncludedProperties != null) {
```

Comment:
Should use a hashset here

Reply:
I've adjusted things to use a hash set in order to check if the schema props should be included.